spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt

```diff
@@ -62,6 +60,23 @@ class SpdxExternalReference(
         OTHER("OTHER");
     }
 
+    // TODO: Using new values as per https://github.com/spdx/spdx-spec/issues/451.
+    enum class Type(
+        private val typeName: String
+    ) {
+        CPE_22_TYPE ("cpe22Type"),
+        CPE_23_TYPE ("cpe23Type"),
+        MAVEN_CENTRAL("maven-central"),
+        NPM("npm"),
+        NUGET("nuget"),
+        BOWER ("bower"),
```

Comment:
Shouldn't we sort this list alphabetically (maybe besides `OTHER`, which should stay last)?